ComponentSnapshotTestCase/CKComponentSnapshotTestCase.h

```diff
@@ -13,10 +13,10 @@
 
 #import <FBSnapshotTestCase/FBSnapshotTestCase.h>
 
-#define CK_AT_LEAST_IOS7_1 (kCFCoreFoundationVersionNumber >= 847.24)
 #define CK_AT_LEAST_IOS8 (kCFCoreFoundationVersionNumber > 847.27)
 #define CK_AT_LEAST_IOS8_2 (kCFCoreFoundationVersionNumber >= 1142.16)
 #define CK_AT_LEAST_IOS9 (kCFCoreFoundationVersionNumber >= 1223.1)
+#define CK_AT_LEAST_IOS10_BETA_4 (kCFCoreFoundationVersionNumber >= 1345.0)
```

Comment:
This will obviously need to be replaced with `CK_AT_LEAST_IOS10` once iOS 10 is released. For now it feels safe enough to base this check on the `kCFCoreFoundationVersionNumber` for beta 4.